README.md

```diff
@@ -1,54 +1,210 @@
+additionally `SetupBundle` & `StateBundle` only for dev environment. To overview, 
+enable or disable  them, you need to edit the file `config/bundles.php` 
+
+
+### Usage of available services 
+
+In every bundle there are services that can be autowired directly in your application. For example
+to autowire `CatalogManager` in a controller action: 
 
 ```php
-class AppKernel extends Kernel
+<?php
+namespace App\Controller;
+
+use Commercetools\Symfony\CatalogBundle\Manager\CatalogManager;
+use Symfony\Component\HttpFoundation\Response;
+
+class MyController
 {
-    public function registerBundles()
+    public function index(CatalogManager $manager)
     {
-        $bundles = [
-            // ...
-            new Commercetools\Symfony\CtpBundle\CtpBundle(),
-            // ...
-        ];
-        // ...
+        /** @var CategoryCollection $categories */
+        $categories = $manager->getCategories('en');
+
+        return new Response(
+            '<html><body>categories: '
+            // handle CategoryCollection $categories
+            .'</body></html>'
+        );
     }
-    // ...
+}
 ```
 
-Open App/config/parameters.yml.dist and add these lines
-```yaml	
-    commercetools.client_id: ~
-    commercetools.client_secret: ~
-    commercetools.project: ~
-```
+The main idea behind is that in each Bundle there are some reusable services that you
+may inject directly in your app. As a generic pattern there are a 
+couple of `*Manager` services, that provide related actions. So, for example in `CartManager`
+you will find helpers like `getCart`, `createCartForUser` and `update` which returns a 
+`CartUpdateBuilder` where you can dynamically set your update actions.
+
+#### Available services
 
-Open App/config/config.yml and these lines on the bottom of the document
-```yaml	
-commercetools:
-  credentials:
-    client_id: "%commercetools.client_id%"
-    client_secret: "%commercetools.client_secret%"
-    project: "%commercetools.project%"
+- CartBundle
+    - CartManager
+    - OrderManager
+    - PaymentManager
+    - ShippingMethodManager
+- CatalogBundle
+    - CatalogManager
+    - Search
+- CtpBundle
+    - ClientFactory
+    - ContextFactory
+    - MapperFactory
+    - LocaleConverter
+    - CustomTypeProvider
+- CustomerBundle
+    - CustomerManager
+    - AuthenticationProvider
+    - UserProvider
+- ReviewBundle
+    - ReviewManager
+- ShoppingListBundle
+    - ShoppingListManager
+- StateBundle
+    - CtpMarkingStoreLineItemState
+    - CtpMarkingStoreOrderState
+    - CtpMarkingStorePaymentState
+    - CtpMarkingStoreProductState
+    - CtpMarkingStoreReviewState
+    
+### Console Commands
+
+To use the console commands navigate to your project's directory and run
+
+```sh
+bin/console commercetools:<command-name>
 ```
 
+#### Available console commands
+
+- SetupBundle
+    - `project-info` // Fetches and displays information from commercetoools platform
```

Comment:
1) full CLI command for all of these, i.e. ` bin/console commercetools:project-info` (make it easy for people to copy and paste)

2) Instead of `//` use `:` so:


> `bin/console commercetools:project-info`: Fetches and displays information from commercetoools platform.